Dockerfile

```diff
@@ -8,20 +8,13 @@ RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 
 # Copy in the go src
 WORKDIR /go/src/github.com/wave-k8s/wave
-COPY Gopkg.lock Gopkg.lock
-COPY Gopkg.toml Gopkg.toml
-
-# Fetch dependencies before copying code (should cache unless Gopkg's change)
-RUN dep ensure --vendor-only
-
-COPY pkg/    pkg/
-COPY cmd/    cmd/
+COPY . .
```

Comment:
To improve caching, I'd suggest copying the `go.mod` and `go.sum` files in, and doing a `go mod download`, then copying everything else in. Assuming the go.mod doesn't change, this should speed up rebuilds

Reply:
Good idea, I don't mind doing that.. will do

Reply:
Done and tested (watched network traffic to ensure it didn't re-download)

- https://github.com/wave-k8s/wave/pull/107/commits/728e447b4ea869e755136234332430593f3e9bef